domains/certificates/chart/values.yaml

```diff
@@ -7,3 +7,14 @@ postgres:
     name: sibedge/postgres-plv8
     tag: 15.0-3.1.4-alpine
   size: 10Gi
+rabbitmq-cluster-operator:
+  extraDeploy:
```

Comment:
Do we want to define the cluster using "extraDeploy"? If possible I would prefer to have it in its own YAML-file in templates/ folder.

Reply:
Marked as resolved, as this pr is still a draft and not done yet.